packages/stock-location/src/utils/build-query.ts

```diff
@@ -0,0 +1,354 @@
+import { ExtendedFindConfig, FindConfig } from "@medusajs/types"
```

Comment:
todo: The buildquery from utils wasn't align with the one in medusa, I ve updated that [here](https://github.com/medusajs/medusa/pull/4148), you can have a look to apply the changes in buildWhere